### DISCUSSION TITLE
Linux 编译版本，点文本换行不生效

### DISCUSSION THREAD

**Discussion by changaolee** [Tencent/libpag#2496] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/c4708333-340f-4634-8d80-b8f9acd702a5)

**Comment by kevingpqi123** [comment]
点文本本身就不支持换行，如果需要换行，使用框文本
https://pag.io/docs/editable-text.html

**Reply by changaolee** [reply]
![image](https://github.com/user-attachments/assets/bd9a107a-b4a4-445a-9831-fc0aeb9fb186)

**Reply by changaolee** [reply]
文档里写的可以在文本中添加 \n 实现，具体是怎么操作呢？

**Reply by changaolee** [reply]
@kevingpqi123 请问是我的理解有误么，这边按照文档说明，在文本中添加了换行符 \n 还是无法换行呢？

**Reply by changaolee** [reply, marked as answer]
这边试了下，用 4.3.62 的代码编译可以正常换行，用 main 分支的会有问题

**Reply by kevingpqi123** [reply]
好，我们定位这个问题
